crates/common/src/entities/player.rs

```diff
@@ -6,3 +6,7 @@ pub fn build_default(builder: &mut EntityBuilder) {
     super::build_default(builder);
     builder.add(Player).add(EntityKind::Player);
 }
+
+/// The hotbar slot a player's cursor is currently on
+#[derive(Debug, Clone, Copy, Eq, PartialEq)]
+pub struct HotbarSlot(pub usize);
```

Comment:
Could we encapsulate the `usize` to enforce that it's always in the range `0..9`? In other words, make the field private and add a setter method that returns a `Result`.

Reply:
Done with d2a0eda